packages/libbinaryen/libbinaryen.105.1.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Libbinaryen packaged for OCaml"
+maintainer: "blaine@grain-lang.org"
+authors: "Blaine Bublitz"
+license: "Apache-2.0"
+homepage: "https://github.com/grain-lang/libbinaryen"
+bug-reports: "https://github.com/grain-lang/libbinaryen/issues"
+depends: [
+  "conf-cmake" {build}
+  "dune" {>= "2.9.1" & < "3.0.0"}
```

Comment:
This is actually incompatible with dune >= 3.0.0 or is just a conservative upper bound?

Reply:
We're being conservative because I haven't put this though the paces. I plan to do that testing and expand the range together with jsoo 4 and the binaryen 106 release. How does that sound?

Reply:
Let's keep it for now. I want to update some tests to use the bin-available macro when we update.